app/assets/javascripts/controllers/ems_keypair/ems_keypair_controller.js

```diff
@@ -0,0 +1,67 @@
+var EmsKeypairController = function($scope) {
```

Comment:
Great work, just wrap whole file in [Immediately-invoked function expressions](http://www.nicoespeon.com/en/2013/05/properly-isolate-variables-in-javascript/), this way variable EmsKeypairController would not be visible in window's scope.